Apps/WebClient/src/ClientApp/src/store/modules/medication/actions.ts

```diff
@@ -57,6 +58,46 @@ export const actions: ActionTree<MedicationState, RootState> = {
             }
         });
     },
+    retrieveMedicationRequests(
+        context,
+        params: { hdid: string }
+    ): Promise<RequestResult<MedicationRequest[]>> {
+        return new Promise((resolve, reject) => {
+            const medicationRequests: MedicationRequest[] =
+                context.getters.medicationRequests;
+            if (context.state.status === LoadStatus.LOADED) {
```

Comment:
This one is not sufficient anymore since the store contains two sources, we need to do it with the partial_loaded

Reply:
The mutation needs to be updated as welll;